.github/workflows/main.yml

```diff
@@ -0,0 +1,70 @@
+name: Run CI checks
+
+on: [push, pull_request]
+
+# By default, jobs run in parallel on a fresh instance of the virtual
+# environment. You can force sequential order by specifying a job's
+# dependencies with the "needs" attribute.
+jobs: 
+
+  # Continuous integration: Run unit tests on all supported platforms whenever
+  # someone pushes a new commit or creates a pull request. 
+  build:
+    name: Building on ${{ matrix.os }}
+    runs-on: ${{ matrix.os }}
+    strategy:
+      max-parallel: 4
+      # Test on all supported platforms using all supported Python versions:
+      matrix:
+        python-version: [3.5, 3.6, 3.7]
+        os: [ubuntu-latest, ubuntu-16.04, windows-latest, macOS-latest]
+    
+    steps:
+    - uses: actions/checkout@master
```

Comment:
Are you sure about this one? Shouldn't this be  `actions/checkout@v1`? See: https://github.com/actions/checkout#usage